azerothcore/azerothcore-wotlk
# fix(DB/Spells): Remove scaling from spell ID 36500

## Changes Proposed:
-  Remove Flag `Scales with caster level` from the Attributes of the Glaive spell (ID 36500) and scale up its base damage and die sides. This prevents the SCRIPT_EFFECT base value from being scaled up and the console won't be spammed with errors. 

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3869

## SOURCE:

None sadly.

## Tests Performed:

- Tested in-game. Windows 10 - enUS client.


## How to Test the Changes:


1. `.go c id 20985`
2. `.q a 10439`
3. Talk to Captain Saeed to start the escort.
4. Observe no errors in the console while the escort adds fight other mobs.

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
